product_docs/docs/biganimal/release/free_trial/detail/create_a_cluster/create_cluster_cli.mdx

```diff
@@ -42,15 +47,15 @@ Switched the context credential to "newuser".
 The command will direct you to open a webpage and copy the randomly generated, one-time code. You’ll need to log in (or already be logged in) to activate the credentials. You can see the credentials you've verified on the command line.
 
 ```shell
-biganimal show-credentials
+biganimal credential show
 __OUTPUT__
-┌──────────────────────────────────────────────────────┐
-│ Credentials                                          │
-├─────────┬─────────────────────────────────────┬──────┤
-│ name    │ address                             │ port │
-├─────────┼─────────────────────────────────────┼──────┤
-│ newuser │ portal.biganimal.com                │ 443  │
-└─────────┴─────────────────────────────────────┴──────┘
+├──────────┬─────────────────────────────────────────────────────./────┐
+│ Credentials                                                        │
+├──────────┬─────────────────────────────┬──────┬────────────────────┤
+│ name     │ address                     │ port │ context credential │
+├──────────┼─────────────────────────────┼──────┼────────────────────┤
+│ new-user │ portal.enterprisedb.network │ 443  │ x                  │
+└──────────┴─────────────────────────────┴──────┴────────────────────┘
```

Comment:
Does the content of the following Caution need adjustments?

> If you add another credential, you’ll need to add `--credential [newuser]` to the following commands. If you have only one, the option isn't needed. You can change the default credential using `biganimal config set context_credential [name]`.

Reply:
If you add another credential, newly created credential will be set as default context_credential. you’ll need to add flag `--credential [newcreds]` to the commands to override the default credentials. If you have only one, the option isn't needed. You can change the default credential using `biganimal config set context_credential [name]`.

Reply:
done